src/processor/FHIRProcessor.ts

```diff
@@ -30,21 +29,16 @@ export class FHIRProcessor {
   }
 
   processConfig(externalDeps?: string[]): ExportableConfiguration {
-    let config: ExportableConfiguration;
     const igForConfig =
       this.lake.getAllImplementationGuides().find(doc => doc.path === this.igPath) ??
       this.lake.getAllImplementationGuides()[0];
-    if (this.lake.getAllImplementationGuides().length > 0) {
-      config = ConfigurationProcessor.process(igForConfig.content);
-    } else {
-      config = ConfigurationExtractor.process(
-        [
-          ...this.lake.getAllStructureDefinitions(),
-          ...this.lake.getAllCodeSystems(),
-          ...this.lake.getAllValueSets()
-        ].map(wild => wild.content)
-      );
-    }
+    const resources = [
+      ...this.lake.getAllStructureDefinitions(),
+      ...this.lake.getAllCodeSystems(),
+      ...this.lake.getAllValueSets()
+    ].map(wild => wild.content);
+    if (igForConfig) resources.push(igForConfig.content);
```

Comment:
I think stylistically we usually avoid one line if statements? Not sure I'm pretty sure @cmoesel, gave me a stern talking to one time for using one. So consider removing this or risk his _wrath_.

Reply:
😳 Do not want to risk his _wrath_ 😳 

(Also pretty sure I've made the same comment before, so I'm risking my own _wrath_)

Reply:
Fixed in 20d75fa!